airbyte-integrations/connectors/source-mixpanel/metadata.yaml

```diff
@@ -18,6 +18,20 @@ data:
     oss:
       enabled: true
   releaseStage: generally_available
+  suggestedStreams:
+    streams:
+      - export
+      - cohorts
+      - cohort_members
+      - engage
+      - annotations
+      - revenue
+      - funnels
+  releases:
+    breakingChanges:
+      1.0.0:
+        message: In this release, the "datetime" column in the "engage" stream will transition from Date/Time/Datetime types to string due to Mixpanel inconsistencies. Additionally, the "credentials" field is now mandatory, and the "project_id" path has been updated, though a config migration will minimize disruptions.
```

Comment:
Nice message. Should the user reset the engage stream as a result of the type change? Maybe we're doing it for them? 

I think you'll need a migrations file - QA checks should let you know this